manifests/install.pp

```diff
@@ -23,9 +23,21 @@
     ensure => $ensure
   }
 
-  package { ['zookeeperd']: #init.d scripts for zookeeper
-    ensure  => $ensure,
-    require => Package['zookeeper']
+
+  if $::lsbdistcodename == 'xenial' {
+      file { '/usr/lib/systemd/system/zookeeper.service':
+          mode    => '0664',
+          source  => 'puppet:///modules/zookeeper/zookeeper.service',
+          require => Package['zookeeper'],
+      } ~> exec { 'systemctl daemon-reload':
+            refreshonly => true,
+            path        => $::path,
```

Comment:
This is strange. You shouldn't need to do that (path)

Reply:
Updated to use systemd::unit_file  where this is not required.